src/styles/index.css

```diff
@@ -436,3 +436,10 @@ html .fabric-editor-popup-scroll-parent > div > div {
     text-transform: none;
     font-weight: normal;
 }
+
+/* Used to avoid conflicts with sharing modal */
+[aria-label="Media floating controls"],
```

Comment:
Shouldn't aria labels be translated?

Is this something forced by the atlassian plugin?